cli.go

```diff
@@ -39,6 +39,7 @@ var (
 	tickerTime        = kingpin.Flag("cc-pull-time", "CloudController Polling time in sec").Default("60s").Envar("CF_PULL_TIME").Duration()
 	requestLimit      = kingpin.Flag("cc-rps", "CloudController Polling request by second").Default("50").Envar("CF_RPS").Int()
 	extraFields       = kingpin.Flag("extra-fields", "Extra fields you want to annotate your events with, example: '--extra-fields=env:dev,something:other ").Default("").Envar("EXTRA_FIELDS").String()
+	cfOrgs            = kingpin.Flag("cf-orgs", "stream only certain cloudfoundry orgs' LogMessage'--cf-orgs=env:dev,something:org1|org2").Default("").Envar("CF_ORGS").String()
```

Comment:
Could be renamed: exclude-orgs.  Description could be about excluding some orgs that need to be comma separated.

Reply:
Agree on this one, cf-orgs just mean nothing, and the description seems wrong `stream only certain cloudfoundry orgs` should not be ``.

So something like `excluded-orgs` desc `Organisation you want to exclude from logs forwading.`